docsrc/admin/installation.md

```diff
@@ -2,85 +2,83 @@
     * To create a bootable CD of the Installation ISO,
       follow [these instructions](https://docs.centos.org/en-US/centos/install-guide/Making_Media/).
-    * Installation Notes
-        * Install as _Server with GUI_ to make it easier to maintain.
-        * When configuring your disk partitions
+    * **Installation Notes**
+        * Select _Server with GUI_ as your Software Selection to make it easier to maintain.
+        * When configuring your disk partitions (Installation Destination), Select Customize Storage Configuration.
             * Use LVM when creating the filesystems. This is the default.
                 * Don't modify /boot or /boot/efi
             * Increase root partition to 100GB
-            * Create a /data partition large enough to hold all of your data now
+            * Create a /data partition large enough to hold all your data now
               and in the future.
+            * Create a /home partition large enough to house your users' data.
             * Leave 10% of the disk unallocated. This will allow you to create
-              snapshots or increase the size of
-              partitions later if necessary.
+              snapshots or increase the size of partitions later if necessary.
+        * Configure the Network and Hostname (and DNS) appropriately for your network.
+        * Set the appropriate Time and Date.
+        * Create a Secure Root Password
+        * Create a user account for the system administrator who will be performing the
+          installation. Under the Advanced configuration, make the user a member of the `wheel` group so they will be able to
+          perform commands as root using `sudo`.
 
-* Create a user account for the system administrator who will be performing the
-  installation.
-  Make that user a member of the `wheel` group so that they will be able to
-  perform commands as root using `sudo`.
-* Install git and python3:
+### Post Operating System Installation
 
-        sudo yum install -y git python3 python3-pip
+Once the server boots up after the installation, you can install the requirements necessary for 
+installing the Data Library Software. You must have **sudo** privileges on your account.
 
-* Install ansible with pip. The `--user` option causes it to be installed in ~
-  /.local, to avoid interfering with python
-  packages provided by CentOS.
+* Disable SELinux:
 
-        # Note: do not use sudo here.
-        python3 -m pip install --user --upgrade pip
-        python3 -m pip install --user ansible==4.5.0
+      sudo sed -i s/SELINUX=enforcing/SELINUX=permissive/ /etc/selinux/config 
+      sudo setenforce permissive
 
-* Disable SELinux and reboot:
+  **Note:** If you feel SELinux is important to the security of your server, please start a conversation with us at help@iri.columbia.edu.
```

Comment:
I have yet to encounter anyone at a national met service who knows what SELinux is, let alone perfers to leave it enabled. Maybe we can remove this note.